### PR TITLE
Add Nodejs 18 support

### PR DESCRIPTION
**Pull request by creativedrewy**
As a note, this makes using node version 16 no longer work. I think I'll add a node version check to the beginning.

### CODE REVIEW

File: packages/core/src/validate/index.ts [hunk 1]
```diff
@@ -55,6 +55,22 @@ export const validateRelease = (releaseJson: MetaplexFileReleaseJsonMetadata) =>
     jsonToValidate.image = jsonToValidate.image?.fileName;
   }
 
+  // Support the replacing of string URIs with a MetaplexFile instance
```
Comment (sdlaver) [inline packages/core/src/validate/index.ts:58]: Can I get a quick summary of why this new code is required for Nodejs 18 support?

**Review by sdlaver — COMMENTED** [review]
After this PR, what range of Nodejs versions are supported (and what have we tested)?

File: packages/core/src/validate/index.ts [hunk 2]
```diff
@@ -55,6 +55,22 @@ export const validateRelease = (releaseJson: MetaplexFileReleaseJsonMetadata) =>
...
+  jsonToValidate.extensions.solana_dapp_store.media.forEach((media, index) => {
+    if (typeof media.uri !== "string") {
+      // @ts-ignore
```
[the remark this answers is not in the document]
Comment (creativedrewy) [reply packages/core/src/validate/index.ts:61]: I'll give you one guess as to which object type is giving us issues

File: packages/core/src/validate/index.ts (hunk 1 above)
Comment (creativedrewy) [reply packages/core/src/validate/index.ts]: So I actually think this particular issue was the result of adding uri to into the schema. I didn't run the app at that time, but I knew I would before cutting an actual new release. So now it came up.

**Review by sdlaver — APPROVED** [review]

**Comment by creativedrewy** [comment]
Fixes #99

**Pull request merged**